.github/workflows/verify_consumer_pacts.yml

```diff
@@ -307,4 +307,4 @@ jobs:
           repo: broadinstitute/terra-github-workflows
           ref: refs/heads/main
           token: ${{ secrets.BROADBOT_TOKEN }} # github token for access to kick off a job in the private repo
-          inputs: '{ "pacticipant": "sam-provider", "version": "${{ needs.verify-consumer-pact.outputs.provider-sha }}" }'
+          inputs: '{ "pacticipant": "sam", "version": "${{ needs.verify-consumer-pact.outputs.provider-sha }}" }'
```

Comment:
sam also can be a consumer, will this cause any name collisions/issues with that?

Reply:
I think ultimately what we want is for sam to self-identify itself as `sam` when acting in both the provider _and_ consumer role.  I think this will let the pact-broker do some dependency graph magic which is currently stymied by having the consumer/provider roles named divergently.

So I think there may be some followup work to rename the consumer side of things as well, but I wanted to try to keep the scope of this change as tight as possible.

Reply:
ok nice that makes sense thanks! btw side-note the verify-consumer-pact action was failing on our pr's and it looks like this change fixed it...?

Reply:
Oh dang, I didn't realize these were failing previously.

And, I hate to disapppoint... but I think that's actually a false 'pass' on this PR...

The first time a pact verification runs for a given provider (in this case, the newly renamed `sam`) it treats the contracts as being in a 'pending' state, and quietly passes the tests (even if they're failing).

You need to go to the [logs](https://github.com/broadinstitute/sam/actions/runs/6694733649/job/18229693055?pr=1234) to see that the tests are still failing with the same looking stack traces as before this PR.

Reply:
Took a quick look at the failure @Ghost-in-a-Jar and I think it's because the `verify_consumer_pacts.yml` needs to somehow run `/bin/bash -c env/test.env` as part of setting up its environment before running sbt test on the `SamProviderSpec`.

I tried a few different incantations (see https://github.com/broadinstitute/sam/pull/1234/commits/5d547ce18d1c5d77097be49553e794a6e73fb359) for latest attempt but kept hitting permission problems, so I am backing that commit out.

Reply:
ok good to know thanks for taking a look!

Reply:
@jladieu I managed to fix it with this change https://github.com/broadinstitute/sam/pull/1233/files#diff-88f8d59210f317b7bdd12e6a807c0b84cf473372221acb3ce860f5a0160183b1